lighthouse.md

```diff
@@ -0,0 +1,66 @@
+# Project Lighthouse
+
+Project Lighthouse is conceptually linked to Project Carmel. They may appear as separate entities, but both projects can potentially share similar technical foundations. While Carmel is focused on debates, Lighthouse takes those principles and applies them to the management of a knowledge base. The underlying architecture can be similar, promoting synergies and allowing both projects to benefit from shared enhancements and updates.
+
+Draft spec, please provide feedback in the form of pull requests and issues to github.com/allinbits/carmel.
+
+## Initial terms
+
+ * All contributors agree on a curator or a panel of curators.
+ * All contributors commit to intellectual honesty, constructive debate, and the pursuit of truth.
+ * All contributors respect that this is an uncensored platform but are responsible for the veracity of their contributions.
+
+## Step 1. Determine the Discussion Points
+
+ * Any contributor can suggest a complex or controversial topic for discussion.
+ * The curator(s) reviews these suggestions for relevance and interest.
+ * Topics that foster meaningful exploration of truth are approved.
+ * The curator(s) can propose new topics independently.
+
+## Step 2. Submission of Arguments and Evidence
+
+ * contributors can submit arguments, counter-arguments, and evidence for each approved topic.
+ * Contributors are expected to cite sources where applicable.
+ * The curator(s) organizes and categorizes the submitted information.
+ * All contributions remain uncensored, but the curator(s) may provide a framework for the information to promote understanding.
+
+## Step 3. Simplification and Correlation
+
+ * Contributors can propose to split complex topics into smaller, easier-to-review subtopics.
+ * Contributors can suggest links to other related topics.
```

Comment:
Here is why I ask. When you mean "_related topics_" you mean topics that are subtopics of the bigger one like in the splitting or can be even non-related in the same big topic? Because obviously this can help to increase the fan of options to provide sources.

Reply:
Establishing intricate/enriched statement relationships allows us to utilize known facts and focus on unraveling the unknown, effectively handling complex projects.